src/erlcloud_ec2.erl

```diff
@@ -1348,37 +1352,73 @@ attribute_xpath(_, AttributeName) ->
     "/DescribeInstanceAttributeResponse/" ++ AttributeName ++ "/value".
 
 
+-type ec2_instances_ids() :: [string()].
+-type ec2_token() :: string() | undefined.
+-type ec2_max_result() :: pos_integer() | undefined.
+-type ok_error(Ok) :: {ok, Ok} | {error, term()}.
+-type ok_error(Ok1, Ok2) :: {ok, Ok1, Ok2} | {error, term()}.
```

Comment:
Well, `ok_error` is always parametrized as either `ok_error(proplist())` or `ok_error(proplist(), ec2_token())`, so I could just roll back to my previous implementation and not parametrize them at all. Doesn't make much difference for dialyzer anyway.